php/views/license.php

```diff
@@ -33,9 +33,9 @@
 						$count = $license['site_count'];
 					}
 
-					$expiry = date( get_option( 'date_format' ) );
+					$expiry = gmdate( get_option( 'date_format' ) );
```

Comment:
This is one of the few changes to a PHP file in this PR, other than `array()` to `[]`.

This appears in 8 other places, and is in response to a PHPCS error.